site/src/components/series/line.md

```diff
@@ -27,3 +48,53 @@ Which gives the following:
 
 This series has the same properties for specifying the x and y value accessors, and decorate as the [point series](#point).
 
+`Line` also exposes the `interpolate` property from the d3 library, which accepts 11 different interpolations:
```

Comment:
I think there are actually 13 types of interpolation - `linear-closed` and `step` are missing from this list.

https://github.com/mbostock/d3/blob/v3.5.6/src/svg/line.js#L83-L98